docs/configure-artifact-repository.md

```diff
@@ -360,36 +360,36 @@ artifacts:
       container: my-container-name
       blob: path/in/container
       # If a managed identity has been assigned to the machines running the
-      # workflow (e.g., https://docs.microsoft.com/en-us/azure/aks/use-managed-identity)
+      # workflow (for example, https://docs.microsoft.com/en-us/azure/aks/use-managed-identity)
       # then useSDKCreds should be set to true. The accountKeySecret is not required
       # and will not be used in this case.
       useSDKCreds: true  
 ```
 
-In addition to managed identities, Argo workflows also support authentication using access keys and SAS tokens.
+You can also authenticate with [Access Keys](#using-azure-access-keys) and [Shared Access Signatures (SAS)](#using-azure-shared-access-signatures-sas).
 
-### Using Azure access keys
+### Using Azure Access Keys
 
```

Comment:
It would be nice to have a link here too for consistency and reference purposes
```suggestion

You can also use an [Access Key](<insert link here>).

```

Reply:
Sorry I don't understand. Why would we add a link to using Access Key under the section of Access Keys?
Maybe you meant we could link to one of the other methods?
Meanwhile, I added a list of links of possible auth methods : 7d8fd2ad6f772d1501cee97be8f4a333d668adc7

Reply:
A link to the Azure docs -- this is the only method missing a link to Azure docs

Reply:
Ahh.. let me try and add that.

Reply:
Done.